tests/test_cli.py

```diff
@@ -20,6 +25,51 @@ def context():
     return ctx().obj["context"]
 
 
+mock_gui_main = """from .app import show
```

Comment:
The problem is that you can't do ```sys.executable, "-m", "mock_gui"``` on python 2.6 https://stackoverflow.com/a/6630931